addons/account/tests/test_account_move_entry.py

```diff
@@ -641,3 +641,19 @@ def test_invoice_like_entry_reverse_caba(self):
                 'credit': value['debit'],
             })
         self.assertRecordValues(reversed_caba_move.line_ids, expected_values)
+
+    def _cache_count(self, field='account.move.name'):
+        for key in self.env.cache._data:
+            if str(key) == field:
+                return len(self.env.cache._data[key])
+        return 0
+
+    def test_cache_invalidation(self):
+        self.env['account.move'].invalidate_cache()
+        lines = self.test_move.line_ids
+        # prefetch
+        lines.mapped('move_id.name')
```

Comment:
Could be simpler, couldn't it?
```suggestion
        self.test_move.name
```

Reply:
My idea is to cache both lines and the move